Lib/threading.py

```diff
@@ -895,6 +895,24 @@ class is implemented.
                 except AttributeError:
                     pass
 
+        # Only check when a program uses pyrona
+        if is_pyrona_program():
+            def movable(o):
+                return isinstance(o, Region) and not o.is_open()
```

Comment:
I guess we should talk about the semantics of `is_open()` at one point. In theory, this will always return true, once the WB is fully in place, since the function call of `movable()` and `o.is_open()` both increase the LRC.

Not a blocker for this PR, but something to consider